docs/Gameplay/Wrought-Iron.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Wrought Iron"
+---
+
+
+# Wrought Iron
+
+Wrought Iron is needed to craft machines starting at High Pressure Steam. It can be obtained by molding Iron Ingots in an Alloy Smelter, smelting the Iron Ingots in a Furnace into Wrought Iron Nuggets and then molding the Wrought Iron Nuggets into an ingots in an Alloy Smelter for further processing, e.g. making Wrought Iron Plates.
```

Comment:
Iron ingots don't need to be molded in an alloy smelter, they're sawed into nuggets and smelted into wrought iron nuggets. The nuggets are then compressed in a steam compressor into wrought iron ingots (since molds require steel, which requires wrought iron)

Reply:
why does steel require wrought iron? making a blast furnace does not required wrought iron.

Reply:
i did not know you could saw ingots into nuggets though

Reply:
> why does steel require wrought iron? making a primitive blast furnace does not require wrought iron.
Maybe I misremember but wrought iron is used to make steel in a PBF

Reply:
i just noticed by reading your comment again that i put "Ingots" in places where it should've been "Nuggets"

Reply:
i just looked it up ingame and you can't saw Iron Ingots into Iron Nuggets, you need an Alloy Smelter

Reply:
Sawing added as of 1.4.5